src/components/TechSpecsSection/TechSpecsSection.tsx

```diff
@@ -0,0 +1,97 @@
+import { Device } from '@/types';
+import React from 'react';
+import styles from './styles.module.scss';
+import { renderRightKeyOfTechSpecs } from '@/utils/renderRIghtKeyOfTechSpecs';
+
+interface Props {
+  product: Device;
+}
+
+export const TechSpecsSection: React.FC<Props> = ({ product }) => {
+  const specsObj = {
+    Screen: product.screen,
+    Resolution: product.resolution,
+    Processor: product.processor,
+    RAM: product.ram,
+    Memory: product.capacity,
+    Camera: product.camera,
+    Zoom: product.zoom,
+    Cell: product.cell,
```

Comment:
I think, we don't need to show this information

![Screenshot 2024-11-10 154625](https://github.com/user-attachments/assets/866a90f6-ac9b-4dfd-9c34-534b959fabeb)